### ISSUE TITLE
Mac Keyboard Navigation doesn't work reliably

### ISSUE THREAD

**Issue by lewing** [xamarin/Xamarin.PropertyEditing#315]
Examples:
- Mock button -> focus "Floating Point:" -> <Tab> -> focus goes to filter entry
- Mock button -> focus "Rectangle:" -> <Tab> <Tab> <Tab> -> focus skips height

**Comment by ermau** [comment]
Let's see if this is a universal editor issue or a per editor issue. If it's per-editor, let's break it out into individual issues.

**Comment by CartBlanche** [comment]
On master it definitely skips straight up to the buttons in the standalone. On dominique-CommonRatio it is better, but there are some improvements that are needed. 

1. we need to remove focus from the up/down buttons so they are skipped over.
2. If the editor for the next control doesn't yet exist, it needs to find the next editor that does exist.

**Comment by CartBlanche** [comment]
Can I close this? I think this issue was addressed in the Common-Ratio PR. If anyone has noticed any other Keyboard navigation issues, let me know please.
